std/range/package.d

```diff
@@ -90,6 +90,14 @@ $(BOOKTABLE ,
     $(TR $(TD $(D $(LREF only)))
         $(TD Creates a _range that iterates over the given arguments.
     ))
+    $(TR $(TD $(D $(LREF padLeft)))
+        $(TD Pads a _range to a specified length by adding a given element to
+        the front of the _range. Is lazy if the range has a known length.
```

Comment:
It's omitting the subject of the sentence (`padLeft`), which is apparently uncommon in English outside of imperative sentences, especially in formal writing. This is probably one reason why we tend towards imperative in function descriptions... and it might be a better idea to use imperative in the module index as well.

Either way I think it's better than repeating the subject verbatim when it's literally the previous word on the page, which is just egregious redundancy. I think it's reasonable to allow for idiosyncratic language in technical manuals and documentation.